### DISCUSSION TITLE
Extend the native Text Panel plugin to include a Rich Text Editor as well?

### DISCUSSION THREAD

**Discussion by ardeora** [grafana/grafana#37660] [Dashboards, Panels & Visualizations]
The native text panel plugin currently has options to add text using HTML and Markdown. It would be a helpful feature for non technical folks to have the ability to edit text using a simple rich text editor. 

<img width="386" alt="Screenshot 2021-08-06 at 4 17 05 PM" src="https://user-images.githubusercontent.com/45807386/128566634-bedd96d7-70f6-4d63-841e-58ec38bc0785.png">

I recently created a custom rich text editor panel plugin for one of our customers and was wondering if this feature would be helpful for other Grafana users as well. I can definitely help extending the component should you decide if it is good idea. Let me know if you have any questions! 😄

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
